backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/database/jpa/PostgresDocumentationUnitHistoryLogRepositoryImpl.java

```diff
@@ -0,0 +1,83 @@
+package de.bund.digitalservice.ris.caselaw.adapter.database.jpa;
+
+import de.bund.digitalservice.ris.caselaw.adapter.transformer.DocumentationOfficeTransformer;
+import de.bund.digitalservice.ris.caselaw.adapter.transformer.HistoryLogTransformer;
+import de.bund.digitalservice.ris.caselaw.domain.DocumentationOffice;
+import de.bund.digitalservice.ris.caselaw.domain.DocumentationUnitHistoryLogRepository;
+import de.bund.digitalservice.ris.caselaw.domain.HistoryLog;
+import de.bund.digitalservice.ris.caselaw.domain.HistoryLogEventType;
+import de.bund.digitalservice.ris.caselaw.domain.User;
+import java.time.Instant;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import org.jetbrains.annotations.Nullable;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class PostgresDocumentationUnitHistoryLogRepositoryImpl
+    implements DocumentationUnitHistoryLogRepository {
+
+  private final DatabaseDocumentationUnitHistoryLogRepository databaseRepository;
+
+  public PostgresDocumentationUnitHistoryLogRepositoryImpl(
+      DatabaseDocumentationUnitHistoryLogRepository databaseRepository) {
+    this.databaseRepository = databaseRepository;
+  }
+
+  @Override
+  public List<HistoryLog> findByDocumentationUnitId(UUID documentationUnitId, @Nullable User user) {
+    return databaseRepository
+        .findByDocumentationUnitIdOrderByCreatedAtDesc(documentationUnitId)
+        .stream()
+        .map(historyLogDTO -> HistoryLogTransformer.transformToDomain(historyLogDTO, user))
+        .toList();
+  }
+
+  @Override
+  public Optional<HistoryLog> findUpdateLogForToday(
```

Comment:
Either: rename method to WithinPeriod or accept one date as param and transform to period in method body